### ISSUE TITLE
no-switch-case-fall-through: Expect break before default

### ISSUE THREAD

**Issue by unional** [palantir/tslint#1033]
Hi, I was expecting `no-switch-case-fall-through` to catch issues like this:

``` ts
switch (prop) {
  case 1:
    // do stuff...
    // forget to break....
  case 2:
    // do stuff...
    break;
  default:
    // do stuff...
    break;
```

But it also catches these:

``` ts
switch (prop) {
  case 1:
  case 2:
    // do stuff...
    break;
  case 3:
  default:
    // do stuff...
    break;
}
```

I think that (especially the `default` case) is a pretty common and valid pattern. What do you think?

**Comment by JKillian** [comment]
You can add a special `/* falls through */` comment, which will tell the rule that you want to allow a fall through to happen. I think 

```
  case 3:
  default:
```

should be allowed though, seems like a bug in the rule potentially.

**Comment by HolgerJeromin** [comment]
I think this is a bug as the comment
https://github.com/palantir/tslint/blob/e5b5ff16677dd00e580839fb1a9ad7602ab114e2/src/rules/noSwitchCaseFallThroughRule.ts#L40
mention it as a valid case.
And perhaps the [test](https://github.com/palantir/tslint/blob/e5b5ff16677dd00e580839fb1a9ad7602ab114e2/test/rules/no-switch-case-fall-through/test.ts.lint#L30) is wrong, too.
